roles/ncn-common/files/scripts/metal/cleanup-live-images.sh

```diff
@@ -95,20 +114,28 @@ else
     echo '-y was present; automatically removing images ...'
 fi
 
-to_remove="$(printf ${BASE}'/%s ' "${LIVE_DIRS[@]}")"
+to_remove_squashfs="$(printf ${SQUASHFS_BASE}'/%s ' "${LIVE_DIRS[@]}")"
+to_remove_overlayfs="$(printf ${OVERLAYFS_BASE}'/%s ' "${LIVE_DIRS[@]}")"
 mount -o rw,remount ${DISK} ${BASE}
-if [ ${ALL} = 1 ]; then
+echo 'Removing squashFS directories ... '
+if [ ${ALL} -eq 1 ]; then
     echo "-a was present; removing ALL images including the currently booted image [${BASE}/${LIVE_DIR}]"
     echo >&2 "This node will be unable to diskboot until it is reimaged with a netboot."
-    rm -rf ${to_remove} "${BASE:?}/${LIVE_DIR}"
+    rm -rf ${to_remove_squashfs} "${SQUASHFS_BASE:?}/${LIVE_DIR}"
```

Comment:
It's almost (entirely?) impossible that these would be all unset, but if they were for some reason it'd be `rm -rf /` ... seems bad.  Same on lines 126 and 130.

Reply:
The `to_remove_` vars?

Reply:
Was talking about all of them.  Just makes me nervous seeing `rm -rf ${vars} / ${vars}`.  I realize it's mostly paranoia. But like what if `lsblk` wasn't found for some reason...

Reply:
Ok, because lines 35-38 I think address this concern..

Reply:
fair